error.go

```diff
@@ -166,6 +166,13 @@ func isLimitExceeded(data string) bool {
 	limitExceedKeys := []string{
 		"limit exceeded",
 		"server too busy",
+		"quota exceeded",
+		"exceeded the maximum allowable",
+		"exceeded your query limit",
+		"restricted due to excessive queries",
+		"due to query limit controls",
+		"you have exceeded you allotted number of",
```

Comment:
Is `you` (in `you allotted number`) a typo in the original server response, or an unintentional one in your PR?

Reply:
Unintentional :(. Thanks, good catch